client/src/components/publications/Publications.js

```diff
@@ -83,7 +84,12 @@ const Publications = () => {
         {loading ? (
           <Spinner animation="border" />
         ) : (
-          <h4>Total of {teamPublications.length} publications</h4>
+          <h4>
+            Total of
+            {teamPublications.length}
+            {' '}
```

Comment:
need a space between `line 88` and `line 89`

Reply:
thanks for the catch